qa/resources/template.conf

```diff
@@ -18,8 +18,9 @@ sparkz {
     bindAddress = "%(API_ADDRESS)s:%(BIND_PORT)s"
     knownPeers = []
     agentName = "2-Hop"
-    maxConnections = %(MAX_CONNECTIONS)d
-    maxPacketSize = %(MAX_PACKET_SIZE)d
+    maxIncomingConnections = %(MAX_INCOMING_CONNECTIONS)d
+    maxOutgoingConnections = %(MAX_OUTGOING_CONNECTIONS)d
+    maxModifiersSpecMessageSize = %(MAX_PACKET_SIZE)d
```

Comment:
`GET_PEERS_INTERVAL` is missing in this file.
So the value specified in `initialize_sc_datadir` is never used.
same for `qa/resources/template_predefined_genesis.conf` file